src/microbe_stage/editor/MicrobeEditor.cs

```diff
@@ -708,13 +725,33 @@ public void SetRigidity(int rigidity)
     }
 
     /// <summary>
-    ///   Removes organelles under the cursor
+    ///   Show options for the organelle under the cursor
     /// </summary>
     [RunOnKeyDown("e_secondary")]
-    public void RemoveOrganelle()
+    public void ShowOrganelleOptions()
     {
         GetMouseHex(out int q, out int r);
 
+        var organelle = editedMicrobeOrganelles.GetOrganelleAt(new Hex(q, r));
+
+        if (organelle == null)
+            return;
+
+        selectedHex = new Hex(q, r);
+
+        gui.ShowOrganelleMenu(organelle);
+    }
+
+    public void OrganelleInProcessOfMoving()
+    {
+        movingOrganelle = editedMicrobeOrganelles.GetOrganelleAt(selectedHex);
```

Comment:
I think the organelle object could go through the gui.ShowOrganelleMenu, so `selectedHex` wouldn't need to be involved in this case.